AIMSICD/src/main/res/values/translatable_strings.xml

```diff
@@ -318,7 +318,7 @@
 
 
     <!-- Section Main -->
-    <string name="help_main_neighboring_cells">Shows neighboring cells.</string>
+    <string name="help_main_neighboring_cells">Shows neighbouring cells.</string>
```

Comment:
@SecUpwN he added new translation file that will create distinction between UK English and American English... Now I wonder how weblate will handle it. Lets see